test/component/cypress/specs/components/Pagination.cy.jsx

```diff
@@ -173,15 +173,15 @@ describe('Pagination', () => {
       cy.window().its('scrollY').should('equal', 0)
     })
   })
-  context('when you navigate to page 2 by clicking the Next link', () => {
+  context('when you navigate to page 2 by clicking the page link', () => {
```

Comment:
```suggestion
  context('when you navigate to page 2 by clicking the Next page link', () => {
```

should that say `Next page` for consistency?